Source/Shared/Library/DefaultCacheConverter.swift

```diff
@@ -37,6 +38,9 @@ public struct DefaultCacheConverter<T> {
 
   /**
    Encodes an instance to NSData
+
```

Comment:
I agree, they are a bit obvious but where would you draw the line? Also, the linter forces you to add documentation.